src/js/file-manager.js

```diff
@@ -163,4 +171,68 @@ fileManager.isTooLarge = () => { return false; };
  */
 fileManager.getMaxSizeReadable = () => { return `${5}MB`; };
 
+/**
+ * @function getMaxImageWidth
+ *
+ * @description Replace with function that determines maximum image width.
+ *
+ * @return {number} maximum image width
+ */
+fileManager.getMaxImageWidth = () => { return 1024; };
+
```

Comment:
get maxPixels from this.props in widget. You can set this to 1024 for now (we'll need a small change in enketo-xslt to get the form-defined value).

Reply:
Done. Now got maxPixels from `filepicker.js` `get props()`